src/Query/Traits/WhereJsonTrait.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+/**
+ * This file is part of Cycle ORM package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Cycle\Database\Query\Traits;
+
+use Cycle\Database\Exception\BuilderException;
+
+trait WhereJsonTrait
+{
+    /**
+     * @param non-empty-string $column
```

Comment:
Need to add comments to all these methods

Reply:
Added